h/cli/commands/groups.py

```diff
@@ -12,8 +12,9 @@ def groups():
 @click.option('--name', prompt=True, help="The name of the group")
 @click.option('--authority', prompt=True, help="The authority which the group is associated with")
 @click.option('--creator', prompt=True, help="The username of the group's creator")
+@click.option('--origin', prompt=True, multiple=True, help="The origin(s) that the group should be scoped to")
```

Comment:
Any value in providing an example `origin` (for formatting reference) here or is it pretty danged obvious how to format an `origin` (or list of same)?

Reply:
Oh, should `origin` be plural to be consistent with later arg names?

Reply:
I added an example. It's plural because you specify the option multiple times, e.g. `./bin/hypothesis --dev groups add-open-group --name MyGroup --authority localhost --creator seanh --origin 'https://example.com' --origin 'https://example2.com'`. With argparse it'd be possible to have the CLI be `--origin` but the variable in the function that receives it be `origins` but I don't think click can do this,